modules/kind/00_mod.mk

```diff
@@ -15,5 +15,7 @@
 images_amd64 ?=
 images_arm64 ?=
 
-images_amd64 += docker.io/kindest/node:v1.27.3@sha256:9dd3392d79af1b084671b05bcf65b21de476256ad1dcc853d9f3b10b4ac52dde
-images_arm64 += docker.io/kindest/node:v1.27.3@sha256:de0b3dfe848ccf07e24f4278eaf93edb857b6231b39773f46b36a2b1a6543ae9
+kind_version := v1.27.3
+
+images_amd64 += docker.io/kindest/node:$(kind_version)@sha256:9dd3392d79af1b084671b05bcf65b21de476256ad1dcc853d9f3b10b4ac52dde
+images_arm64 += docker.io/kindest/node:$(kind_version)@sha256:de0b3dfe848ccf07e24f4278eaf93edb857b6231b39773f46b36a2b1a6543ae9
```

Comment:
question: From previous discussions, I remember Kind being a bit weird with container images. The [latest release](https://github.com/kubernetes-sigs/kind/releases/tag/v0.20.0) says explicitly that the exact images listed in the release must be used.

We've had issues with this in cert-manager too.

Does this script automatically update kind images using crane? If so, is it possible / desirable for us to not do that?

Reply:
Thanks for the pointer, I did not realise kind has its own special way of tagging images.
I'll try to not update the kind images in the script.

Reply:
I added an exception for `docker.io/kindest/node` to the script.